test/spec/controllers/eventDetails.js

```diff
@@ -1,65 +0,0 @@
-'use strict';
```

Comment:
This happened before I completely recreated local environment with Git history of deleting this file.  So I recreated the file and pushed branch.